modules/sharedaddy/sharing-sources.php

```diff
@@ -711,7 +719,13 @@ public function has_custom_button_style() {
 	}
 
 	public function display_footer() {
-		$this->js_dialog( $this->shortname, array( 'height' => 350 ) );
+		if ( ! $this->smart ) {
```

Comment:
Minor nitpick, but when doing an `if` `else`, it's easier to read when the initial condition does not include the `!`. In other words, I'd find it easier to read if this were rearranged as:

```php
if ( $this->smart ) {
	?>
	<script>!function(d,s,id){var js,fjs=d.getElementsByTagName(s)[0],p=/^http:/.test(d.location)?'http':'https';if(!d.getElementById(id)){js=d.createElement(s);js.id=id;js.src=p+'://platform.twitter.com/widgets.js';fjs.parentNode.insertBefore(js,fjs);}}(document, 'script', 'twitter-wjs');</script>
	<?php
} else {
	$this->js_dialog( $this->shortname, array( 'height' => 350 ) );
}
```

(In fact, the only reason I mention this now is because I thought it was a bug that we were not outputting the `<script>` tag when `$this->smart` was in effect, but only because I misread the condition)